text/0000-project-groups.md

```diff
@@ -0,0 +1,328 @@
+around a particular feature or project in the in the organisation, and use this
+space to discuss and iterate on that feature.
+
+Part of building a community is removing some of the institutional memory that
+develops in the design process, and centralising the information and discussion
+around the feature so that we can provide better visibility into why certain
+decisions and trade offs were made over others.
+
+Previously a lot of the discussion and iteration for large features would
+happen in the initial RFC thread. This leads to a lot of discussion in the top
+of the thread and that often becomes completely irrelevant to the current
+iteration.
+
+This process has also been unsuitable to describe features that can take
+multiple years to develop and will become multiple RFCs over the course of its
+design process. Some examples of of this are the "`impl Trait`" and "macros
+2.0" features, where the goals has shifted a lot from the initial RFCs, and it
+can be hard to know their current status.
+
+[ffi unwind]: https://github.com/rust-lang/project-ffi-unwind
+[inline asm]: https://github.com/rust-lang/project-inline-asm
+[safe transmute]: https://github.com/rust-lang/project-safe-transmute
+[triage]: https://github.com/rust-lang/team/blob/master/teams/wg-triage.toml
+[rustup]: https://github.com/rust-lang/team/blob/master/teams/wg-rustup.toml
+
+## Creation
+
+Since project groups are approved by their relevant parent team over the core
+team, it's up to each team decide their specific requirements. However the author
+recommends that a group should try to address at least of the following questions.
+
+- What value do you see your group bringing to the organisation?
+- What are the goals of your group?
+  - Both in the short term, and if relevant over a longer period.
+- What are explicitly non-goals of your group?
+
+These first three points should also serve as the basis for the charter.
+
+- Is your group long-running or temporary?
+- If it is temporary, how long do you see it running for?
+  - What is the long-term vision of your group?
+- What do you expect the relationship to the organisation be?
+- How do you want to establish accountability?
+- If applicable, which other groups or teams do you expect to have close
+  contact with?
+- Who are the initial shepherds/leaders? (This is preferably 2–3 individuals,
+   but not required.)
+- How do you intend to make your work accessible to people outside your group?
+- Where do you see your group needing help?
+
+Process around project group membership is up to the shepherd's discretion.
+Typically, people who are productively contributing to the project group for
+some time will be added as members. It is not required for a project group to
+have a lot of members though, some project groups may only have one or
+two members.
+
+**A project group should have the following;**
+
+- A charter that defines the scope and intent of the group.
```

Comment:
I think some of the questions above should be required; obviously in the end it's up to the team accepting a project group but I would want a good portion of them to always be included. We should say which ones those are; I don't think this is a complete list, but at least:

* What value do you see your group bringing to the organisation?
* What are the goals of your group?
  * Both in the short term, and if relevant over a longer period.
* What are explicitly non-goals of your group?
* What do you expect the relationship to the organisation be?
* Who are the initial shepherds/leaders? (This is preferably 2–3 individuals, but not required.)
* How do you intend to make your work accessible to people outside your group?

Reply:
I've reworded the list of questions so that a charter should try to address all of the questions. As I feel the seperation between required and optional is not really valuable.